src/PureCloud.Client/Apis/LanguagesApi.cs

```diff
@@ -85,10 +166,25 @@ public async Task<Dictionary<string, object>> GetLanguagesTranslationsUserAsync(
 
         var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
 
-        var response = await client.GetAsync($"api/v2/languages/translations/users/{userId}", cancellationToken);
+        var response = await client.GetAsync($"api/v2/languages/translations/users/{Uri.EscapeDataString(userId)}", cancellationToken);
 
         response.EnsureSuccessStatusCode();
 
         return await response.Content.ReadFromJsonAsync<Dictionary<string, object>>(_options.JsonSerializerOptions, cancellationToken);
     }
-}
\ No newline at end of file
+
+    /// <inheritdoc />
+    [Obsolete("This endpoint is deprecated. Please see the Routing API")]
+    public async Task<Language> PostLanguagesAsync(Language body, CancellationToken cancellationToken = default)
+    {
+        ArgumentNullException.ThrowIfNull(body, nameof(body));
```

Comment:
The second argument is not needed and should be removed. Please apply the same thing everywhere